agent/consul/client_test.go

```diff
@@ -219,15 +224,13 @@ func TestClient_RPC_Pool(t *testing.T) {
 }
 
 func TestClient_RPC_ConsulServerPing(t *testing.T) {
+	t.Parallel()
 	var servers []*Server
 	var serverDirs []string
 	const numServers = 5
 
-	for n := numServers; n > 0; n-- {
-		var bootstrap bool
-		if n == numServers {
-			bootstrap = true
-		}
+	for n := 0; n < numServers; n++ {
+		bootstrap := n == 0
```

Comment:
In addition, the loop was also running in reverse from 5 down to 0. This is simpler IMO w/o changing the semantics. In essence, you only want one server with the `boostrap` flag set. Which one doesn't matter.